source/_integrations/tuya.markdown

```diff
@@ -67,6 +67,8 @@ Tuya Smart app. You can find it by opening the app and:
 
 {% include integrations/config_flow.md %}
 
+{% important %} If the integration does not ask you for a User Code but instead asks you for Tuya IoT credentials, then you're most likely using an outdated version of Home Assistant. Please update your instance to use the latest Tuya integration. {% endimportant %}
```

Comment:
Our documentation always reflects the current stable version of Home Assistant.

Essentially, you could write this warning on every line of every single piece of documentation of Home Assistant.

Additionally, one can run custom integration. More often, these are results of people overriding the internals of Home Assistant that way (which is already unsupported).

For the above reasons, I'm going to decline this PR.

../Frenck

Reply:
Fair enough, I just found it easier to put a warning since there was a massive change in the integration configuration. I found myself not understanding what was wrong until I got to the documentation feedback that has 3 issues related to the point I'm making. 

I don't run a custom integration, I just don't have a way to automatically update Home Assistant when every stable version comes out which might as well be most of the cases imho.

But all good, thanks for looking at it 🤗